### PR TITLE
chore: optimize dts imports

### PR DESCRIPTION
**Pull request by DDDDD12138**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

I noticed that the `typings/components.d.ts` file is almost identical to the `global.d.ts` file. The `global.d.ts` file is only copied to the `dist/element-plus` folder after the build process and has no other use. 
Therefore, I propose merging these two files into one. This will also prevent issues where type references might be missing in `global.d.ts`, causing automatic imports to fail.

#### Before
before
#### After
after2

### CODE REVIEW

**Review by tolking — APPROVED** [review]
LGTM

File: typings/global.d.ts [hunk 1]
```diff
@@ -101,6 +99,8 @@ declare module 'vue' {
     ElWatermark: typeof import('element-plus')['ElWatermark']
     ElTour: typeof import('element-plus')['ElTour']
     ElTourStep: typeof import('element-plus')['ElTourStep']
+    ElAnchor: typeof import('element-plus')['ElAnchor']
+    ElAnchorLink: typeof import('element-plus')['ElAnchorLink']
```
Comment (btea) [inline typings/global.d.ts:103]: Why have these two lines of code been moved here?

Comment (DDDDD12138) [reply typings/global.d.ts:103]: I used git mv to rename the `typings/components.d.ts` file to `typings/global.d.ts`, but for some reason, github is comparing it with the `global.d.ts` file in the parent directory.

Comment (tolking) [reply typings/global.d.ts:103]: Perhaps we can adjust the order of the ElAnchor and ElAnchor (L102,L103 -> L9,L10)

Comment (DDDDD12138) [reply typings/global.d.ts]: ok, I’ve updated it.

**Review by btea — APPROVED** [review]

**Comment by DDDDD12138** [comment]
It seems like it hasn’t been merged yet. What else do I need to do?

**Review by warmthsea — APPROVED** [review]
LGTM.

**Pull request merged**